GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -514,9 +512,8 @@ public void run() {
 
                 //For each group, write the repeats from top to bottom
                 for (Entry<QuestionGroupDto, List<QuestionDto>> groupEntry : questionMap.entrySet()) {
-                    Long gid = groupEntry.getKey().getKeyId();
                     if (safeTrue(groupEntry.getKey().getRepeatable())) {
-                        writeInstanceDataSplit(qgSheetMap.get(gid),
+                        writeInstanceDataSplit(qgSheetMap.get(groupEntry.getKey()),
```

Comment:
The previous version of this code, `gid` was a `Long` and now we use `Key` as value. Is this correct?

Reply:
Yes. It tests ok. That Long was the only remnant of the old qgSheetMap keys, which were the QuestionGroup key ids. But there are none for a published form. So qgSheetMap was changed to heve QuestionGroupDto's as keys. In that one place it was not changed.